kafka/src/main/java/io/micronaut/configuration/kafka/ConsumerRegistry.java

```diff
@@ -72,11 +74,19 @@
     /**
      * Is the consumer with the given ID paused.
      *
-     * @param id True it is paused
+     * @param id the consumers id
      * @return True if it is paused
      */
     boolean isPaused(@NonNull String id);
 
+    /**
+     * Is the consumer with the given ID paused to consume from the given topic partitions.
+     *
+     * @param id the consumers id
+     * @return True if it is paused for the topic partitions
+     */
+    boolean isPaused(@NonNull String id, Collection<TopicPartition> topicPartitions);
```

Comment:
Can you add `@NonNull` or `@Nullable` to the second arg and add `@since` to the javadoc

Reply:
I can do that, what should @since contain?